libs/platform/experience/src/services/layout/plugins/styles/image/index.ts

```diff
@@ -0,0 +1 @@
+export * from './image-style.model';
```

Comment:
Are we not exporting the schemas and plugin files from the plugins? How to extend them then?

Reply:
I forgot. We need to add it to static exports (`static.ts`)